### PR TITLE
PLAT-932 Remove redundant systemName field

### PR DESCRIPTION
**Pull request by daniel-klose**

### CODE REVIEW

File: platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java [hunk 1]
```diff
@@ -200,6 +200,14 @@ public interface CoreI18n extends EmfI18n {
...
 	I18n0 DAYS_SINCE_LAST_LOGIN = new I18n0("Days Since Last Login")//
 		.de("Tage seit letzten Einloggen");
+	I18n0 DBMS_CONFIGURATION = new I18n0("Dbms Configuration")//
+		.de("Dbms-Konfiguration");
+	I18n0 DBMS_CONFIGURATION_LOG = new I18n0("Dbms Configuration Log")//
+		.de("Dbms-Konfigurations-Log");
+	I18n0 DBMS_CONFIGURATION_LOGS = new I18n0("Dbms Configuration Logs")//
+		.de("Dbms-Konfigurations-Logs");
+	I18n0 DBMS_CONFIGURATIONS = new I18n0("Dbms Configurations")//
+		.de("Dbms-Konfigurationen");
```
Comment (oliver-richers) [inline platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java:210]: in natural languages, abbreviations are usually all uppercase, AFAIK
```suggestion
	I18n0 DBMS_CONFIGURATION = new I18n0("DBMS Configuration")//
		.de("DBMS-Konfiguration");
	I18n0 DBMS_CONFIGURATION_LOG = new I18n0("DBMS Configuration Log")//
		.de("DBMS-Konfigurations-Log");
	I18n0 DBMS_CONFIGURATION_LOGS = new I18n0("DBMS Configuration Logs")//
		.de("DBMS-Konfigurations-Logs");
	I18n0 DBMS_CONFIGURATIONS = new I18n0("DBMS Configurations")//
		.de("DBMS-Konfigurationen");
```

File: platform-core-module/src/main/java/com/softicar/platform/core/module/AGCoreModuleInstance.java [hunk 2]
```diff
@@ -25,6 +25,11 @@ public static AGCoreModuleInstance getInstance() {
 		return AGCoreModuleInstance.TABLE.get(SINGLETON_INSTANCE_ID);
 	}
 
+	public static String getSystemIdentifier() {
```
Comment (oliver-richers) [inline platform-core-module/src/main/java/com/softicar/platform/core/module/AGCoreModuleInstance.java:28]: hmm, why is the method called getSystemIdentifier() but the field is called SYSTEM_NAME? Should not both be equal? Though, I have no preference for either one.

**Review by oliver-richers — CHANGES_REQUESTED** [review]

File: platform-core-module/src/main/java/com/softicar/platform/core/module/AGCoreModuleInstance.java (hunk 2 above)
Comment (daniel-klose) [reply platform-core-module/src/main/java/com/softicar/platform/core/module/AGCoreModuleInstance.java:28]: its called like that cause I didnt change it from `AGLiveSystemConfiguration` :D

Comment (oliver-richers) [reply platform-core-module/src/main/java/com/softicar/platform/core/module/AGCoreModuleInstance.java:28]: hmpf, well

**Pull request merged**